index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" />
+    <title>Sergio Usma's portfolio</title>
+  </head>
+  <body>
+    <div class="header">
+      <header>
+        <nav>
+          <ul>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+
+          <div class="burger">
+            <div class="burger-line top"></div>
+            <div class="burger-line mid"></div>
+            <div class="burger-line bot"></div>
+          </div>
+        </nav>
+      </header>
+    </div>
+
+    <section id="main">
+      <div class="main">
+        <h2>
+          Hey there. <br> I'm Sergio. <br />
+          <span>I'm a software <br>developer</span>
+        </h2>
+
+        <p>
+          I can help you build a product, feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hesitate to contact me.
+        </p>
+
+        <div class="main__social-media">
+          <ul>
+            <li>
+              <a href="#"><img src="./assets/ico/twitter.png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./assets/ico/linkedin.png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./assets/ico/medium.png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./assets/ico/github.png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./assets/ico/angellist.png" alt="" /></a>
+            </li>
+          </ul>
```

Comment:
- [ ] Please make sure that `alt` attribute is not empty. `alt` attributes are helpful for accessibility reasons, which means to help participants with physical or vision impairments to navigate your site 💪 Please consider adding the appropriate and relevant text for it 📌
- [OPTIONAL] I think it would be better if you add the links of your actual social profiles ✨ This is a great way to build a network. What do you think about it? 💭